internal/mempool/mempool_test.go

```diff
@@ -3254,3 +3254,75 @@ func TestSubsidySplitSemantics(t *testing.T) {
 	}
 	testPoolMembership(tc, postDCP0010Vote, false, true)
 }
+
+// MaybeAcceptTransactions attempts to add a collection of transactions provided
```

Comment:
test name is `TestMaybeAcceptTransactions`, not `MaybeAcceptTransactions`.

Reply:
updated